packages/core/src/common/types.ts

```diff
@@ -39,3 +40,46 @@ export type DataModuleSession = {
 export type Session = {
   close: () => void;
 };
+
+export type AudioPlayerConfig = {
+  isMuted: boolean;
+  isPlaying: boolean;
+  maxVolume: number;
+  severity: number | undefined;
+  soundID: number | undefined;
+  player: Howl | undefined;
+  localDev: boolean;
+};
+
+export interface AudioPlayer {
+  readonly config: AudioPlayerConfig;
+  isPlaying(): boolean;
+  isMuted(): boolean;
+  mute(): void;
+  unmute(): void;
+  play({ severity, volume, audioSrc }: { severity: number; volume: number; audioSrc?: string }): boolean;
+  stop(): void;
+  setMaxVolume(maxVolume: number): void;
+  getMaxVolume(): number;
+}
+
+export type AudioAlertNodeConfig = {
+  audioAlertPlayer: AudioPlayer;
+  isMuted: boolean;
+  volume: number;
+  severity: number;
+  player: Howl | undefined;
+  soundID: number | undefined;
+  audioSrc: string | undefined;
+};
+
+export interface AudioAlertNode {
```

Comment:
Is there any specific reason to call this a `Node`? If not, I think it would be more clear and simple to call this `AudioAlert`